data/random-teams.js

```diff
@@ -410,8 +410,8 @@ class RandomTeams extends Dex.ModdedDex {
 		];
 
 		// Iterate through all moves we've chosen so far and keep track of what they do:
-		for (let k = 0; k < moves.length; k++) {
-			let move = this.getMove(moves[k]);
+		for (const [k, moveId] of moves.entries()) {
```

Comment:
why `entries()`? is `k` actually used here?

Reply:
yeah, it's really long so you cant find it easily but near the end of the loop it calls for ``moves.splice(k, 1);``

Reply:
I should probably mention i'm fine with removing `.entries()` and adding `moves.indexOf(moveId)` where k is used independently, but as stated before, precedent